nengo/probe.py

```diff
@@ -28,9 +30,10 @@ def validate(self, probe, attr):
                                 "into 'weights'. Probe 'weights' instead.",
                                 since="v2.1.0")
         if attr not in probe.obj.probeable:
-            raise ValidationError("Attribute %r is not probeable on %s.\n"
+            raise ValidationError("Attribute %r is not probeable on %s.%s"
```

Comment:
Do you have a Windows command line to verify that `'\n'` works? I think back in the days it would print `'one\ntwo'` as
```
one
   two
```